packages/block-editor/src/components/block-list/block-contextual-toolbar.js

```diff
@@ -9,13 +9,14 @@ import { __ } from '@wordpress/i18n';
 import NavigableToolbar from '../navigable-toolbar';
 import { BlockToolbar } from '../';
 
-function BlockContextualToolbar( { focusOnMount } ) {
+function BlockContextualToolbar( { focusOnMount, ...props } ) {
 	return (
 		<NavigableToolbar
 			focusOnMount={ focusOnMount }
 			className="block-editor-block-contextual-toolbar"
 			/* translators: accessibility text for the block toolbar */
 			aria-label={ __( 'Block tools' ) }
+			{ ...props }
```

Comment:
Why do we need this?

Reply:
It's used to pass down `data-align` (was missing from a bad rebase). This will no longer be needed when we move the movers into the toolbar.